assets/training/finetune_acft_hf_nlp/src/model_converter/model_converter_adapters.py

```diff
@@ -229,20 +237,62 @@ def __init__(self, component_args: Namespace) -> None:
         super().__init__()
         super(ModelConverter, self).__init__(component_args)
 
+    @staticmethod
+    def _remove_unwanted_packages(model_save_path: str):
+        """Remove unwanted packages from conda and requirements file"""
+        if is_main_process():
+            req_file_path = os.path.join(model_save_path, "requirements.txt")
+            conda_file_path = os.path.join(model_save_path, "conda.yaml")
+            requirements = None
+            if os.path.exists(req_file_path):
+                with open(req_file_path, "r") as f:
+                    requirements = f.readlines()
+                if requirements:
+                    for package in UNWANTED_PACKAGES:
+                        requirements = [item for item in requirements if not item.startswith(package)]
+                    logger.info("Updated requirements.txt file")
+
+            conda_dict = None
+            if os.path.exists(conda_file_path):
+                with open(conda_file_path, "r") as f:
+                    conda_dict = yaml.safe_load(f)
+                if conda_dict is not None and "dependencies" in conda_dict:
+                    for i in range(len(conda_dict["dependencies"])):
+                        if "pip" in conda_dict["dependencies"][i] and isinstance(conda_dict["dependencies"][i], dict):
+                            pip_list = conda_dict["dependencies"][i]["pip"]
+                            if len(pip_list) > 0:
+                                for package in UNWANTED_PACKAGES:
+                                    pip_list = [item for item in pip_list if not item.startswith(package)]
+                                conda_dict["dependencies"][i]["pip"] = pip_list
+                                break
+                    with open(conda_file_path, "w") as f:
+                        yaml.safe_dump(conda_dict, f)
+                    logger.info("Updated conda.yaml file")
+
+
+    def _t5_model_on_text_classification_condition_meets(self, model_type):
+        """Check for t5 text-classification"""
+        return self.mlflow_task_type == "text-classification" and model_type == 't5'
```

Comment:
Can you please import `text-classification` and `t5` from constants?

Reply:
Thanks @jpmann , updated